### DISCUSSION TITLE
请问同时选中几个节点的时候，是否可以做到同时缩小放大这些节点？

### DISCUSSION THREAD

**Discussion by ElsaOOo** [antvis/X6#1972] [Q&A]
同时选中几个节点的时候，是否可以做到同时缩小放大这些节点？

**Comment by georgezhu08** [comment]
应该可以自己写出来，只是实现起来比较麻烦，我能想到的办法就是监听node:resizing，然后根据变化取出来其他选中的节点，代码大致应该是这个样子的：

graph.on('node:resizing', ({e, x, y, node, view}) => {
&nbsp;&nbsp;&nbsp;&nbsp; const selectedCells = graph.getSelectedCells();
&nbsp;&nbsp;&nbsp;&nbsp; selectedCells.forEach((cell) => {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  if (cell.isNode() && cell !== node) {
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   // 处理这个选中的节点
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  }
&nbsp;&nbsp;&nbsp;&nbsp; })
})

这里的主要问题是，在node:resizing的时候拿到的是鼠标的当前坐标位置，而不是变化量，所以还需要自己计算出实际变化量是多少，再根据需要去缩放其他节点。

**Reply by z7x0** [reply]
请问有办法拿到框选的框吗  想对框 操作 来实现同时缩放节点
